### ISSUE TITLE
Скрітие пунктов меню

### ISSUE THREAD

**Issue by zivio** [fluxuator/dune_plugin_menu_manager#10]
Можно ли добавить скрытие пунктов меню  в ленточном интерфейсе:
Решение по скрытию лишних пунктов меню есть.
1. С папки /firmware/config копируем в  в /config menu_config.xml
2. редактируем menu_config.xml  в /config- удаляем/комментируем лишние пункты меню.
3. В консоли killall shell или перегружаем плеер.

**Comment by fluxuator** [comment]
Да, я в курсе как это работает. Сейчас разрабатываю более надежный обработчик этого конфига, так как обнаружилось, что даже минимальное повреждение или несоответствие конфига приводит к невозможности загрузить интерфейс плеера. Это можно увидеть только в shell.log. 

Поэтому это конфиг после изменений всегда обязан быть валидным. Иначе, плеер можна будет оживить только удалив menu_config.xml из директории конфигов. Думаю, ты сам понимаешь, какой будет поток негатива от пользователей, которые не способны будут исправить это проблему самостоятельно.

**Comment by zivio** [comment]
а как можно сделать его не валидным плагиногм? Это же не ручная обработка что чтото не так сделал. Допустим считали xml  удалили что не надо сохранили xml. Как он может стать невелидным?
Просто возможно должна быть защита от дурака на пример если нельзя удалять все пункты + сброс (попросту удаление /config/config menu_config.xml)

Как мне кажется в плагине сложно напортачить если сохранять именно xml или даже как рекстовый файл обычным file_put_contents

**Comment by fluxuator** [comment]
Невалидным плагином? Я говорю о конфигурационном файле. Повредить можно как угодно: начиная от отключения питания во время записи, заканчивая ошибкой в программе которая формирует этот файл. У меня это удавалось несколько раз.

**Comment by zivio** [comment]
Как можно сделать конфигурационном файл невалидным если редактировать его плагином.  Там как по мне ошибиться сложно. 
А отключение питания это да.  ИМХО надо проверить сброс настроек через флешку исправляет проблему или нет. Если да то на пример при входе в раздел редактирования меню сделать предупреждение.
